internal/service/encryptionatrest/resource_test.go

```diff
@@ -193,17 +178,9 @@ func TestAccEncryptionAtRest_basicAzure(t *testing.T) {
 	})
 }
 
-func testCheckResourceAttr(resourceName, prefix string, attrsMap map[string]string) resource.TestCheckFunc {
-	checks := acc.AddAttrChecksPrefix(resourceName, []resource.TestCheckFunc{}, attrsMap, prefix)
-
-	return resource.ComposeAggregateTestCheckFunc(checks...)
-}
-
 func TestAccEncryptionAtRest_azure_requirePrivateNetworking_preview(t *testing.T) {
-	acc.SkipTestForCI(t) // needs Azure configuration
-
 	var (
-		projectID = os.Getenv("MONGODB_ATLAS_PROJECT_ID")
+		projectID = os.Getenv("MONGODB_ATLAS_PROJECT_EAR_PE_ID")
```

Comment:
Created new Atlas project `test-acc-tf-p-keep-ear-private-endpoint` with preview flag enabled for this feature